pkg/errox/roxerror.go

```diff
@@ -108,7 +119,8 @@ func (e *RoxError) CausedBy(cause interface{}) error {
 // provided format specifier and arguments, following a colon.
 //
 // Example:
-//     return errox.InvalidArgument.CausedByf("unknown parameter %v", p)
+//
+//	return errox.InvalidArgument.CausedByf("unknown parameter %v", p)
```

Comment:
General remark for code blocks in comments:
if an empty line comes after code blocks to separate from other comment content, I think an empty comment line should come after code blocks in any case (including comment end).
If the style checker does not agree, then let's keep these comments in the current updated state.

Reply:
No, it works fine. I think empty line might be needed to end block and start text or list
![image](https://user-images.githubusercontent.com/1616386/216098076-4479d126-ff03-4c20-8f0c-4345c3a9f83a.png)